src/Microsoft.DotNet.ApiSymbolExtensions/AssemblySymbolLoader.cs

```diff
@@ -39,10 +39,11 @@ public class AssemblySymbolLoader : IAssemblySymbolLoader
         public const string AssemblyReferenceNotFoundErrorCode = "CP1002";
 
         /// <inheritdoc />
-        public AssemblySymbolLoader(bool resolveAssemblyReferences = false)
+        public AssemblySymbolLoader(bool resolveAssemblyReferences = false, bool includeInternals = false)
```

Comment:
Please remove the inheritdoc above (which is wrong) and document the ctor with its parameters?